subprojects/core/src/main/java/org/gradle/tooling/provider/model/internal/DefaultIntermediateToolingModelProvider.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.gradle.plugins.ide.internal.tooling;
+package org.gradle.tooling.provider.model.internal;
```

Comment:
ℹ️  This is a more natural place for the model provider implementation, and is also required for Kotlin DSL builders that do not depend on the `ide` subproject.